packages/mana/consensusbasevector_test.go

```diff
@@ -653,3 +654,91 @@ func TestConsensusBaseManaVector_ToAndFromPersistable(t *testing.T) {
 	}
 	assert.Equal(t, bmv.(*ConsensusBaseManaVector).vector, restoredBmv.(*ConsensusBaseManaVector).vector)
 }
+
+func TestConsensusBaseManaVector_BuildPastBaseVector(t *testing.T) {
+	bmv, err := NewBaseManaVector(ConsensusMana)
+	assert.NoError(t, err)
+	var eventsLog []Event
+	emptyID := identity.ID{}
+
+	tx1Info := &TxInfo{
```

Comment:
As I understand, the idea here is to have 3 transactions.
 - Tx1 spends 10 iotas from genesis output.
 - Tx2 spends 5 iotas from Tx1 outputs.
 - Tx3 spends 5 iotas from Tx1 outputs, 5 iotas from Tx2 output.

Assuming above scenario, `txInfo` definitions should be modified:
 - Tx1 can stay the same.
 - Tx2 timestamp should be `txTime+delta`, Tx2 `InputInfo` timestamp should be `txTime`, as it corresponds to the output created by Tx1.
 - Tx3 timestamp should be `txTime+2*delta`, `InputInfo` timestamps modified to correspond to Tx1 and Tx2 timestamps.